game/src/game.rs

```diff
@@ -14,12 +16,11 @@ impl Plugin for GameplayPlugin {
         // All LDTK stuff should be defined here,
         // so it is all in one place and easy to change!
         // Don't scatter it across the sub-modules/plugins!
-        app.register_ldtk_int_cell::<wall::WallBundle>(17);
+        app.register_ldtk_int_cell::<wall::WallBundle>(17)
+            .register_ldtk_entity::<PlayerBlueprintBundle>("Player")
+            .insert_resource(Gravity::default());
```

Comment:
Style: I would prefer if you split this into several statements rather than use method chaining / builder syntax.

Reply:
Several statements do make it much easier to remove/comment things like this out. Will stick to that in future.